person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
```

Comment:
- [ ] Since :name and :age instance variables would be modified in the `Person class`, we need to define 
 both Getter and Setter methods  for them.  Something like so: 
```
attr_accessor :name, :age
```
- [ ] We need to define a `Getter method` for :id variable to be able to retrieve it's value. Something like so: 
```
attr_reader :id
```